lib/adyen/client.rb

```diff
@@ -160,7 +160,7 @@ def call_adyen_api(service, action, request_data, headers, version, with_applica
             raise connection_error, "Connection to #{url} failed"
           end
         end
-      else
+       if action.fetch(:method) == 'POST'
```

Comment:
yet it's always POST

Reply:
So this POST is created for automated ones only, because later you have another else that does the POST request. The reason for that is that manually created services would pass a URL only without the method, which would always mean POST, but the generated ones always pass a hash with endpoint and a method. Once we have everything generated this will be improved.